x-pack/plugins/observability_ai_assistant/public/service/create_chat_service.ts

```diff
@@ -174,17 +174,20 @@ export async function createChatService({
           });
       });
     },
-    chat({
-      connectorId,
-      messages,
-      function: callFunctions = 'auto',
-      signal,
-    }: {
-      connectorId: string;
-      messages: Message[];
-      function?: 'none' | 'auto';
-      signal: AbortSignal;
-    }) {
+    chat(
```

Comment:
Is this even used anywhere?

Reply:
it's not, I think, but I want to keep it there for now to allow consumers to use one-off interactions with the Assistant (where they can define their own functions and what not)